packages/web-components/src/components/va-pagination/test/va-pagination.e2e.ts

```diff
@@ -100,13 +100,13 @@ describe('va-pagination', () => {
     expect(pageNumbers[pageNumbers.length - 1].innerHTML).toEqual('6');
   });
 
-  it('renders all page numbers if total pages is less than or equal to 7', async () => {
+  it('renders all page numbers if total pages is less than or equal to 6', async () => {
```

Comment:
There might be a test for this already but can you double check if there's one that validates it doesn't exceed 7 slots?